orchagent/portsorch.cpp

```diff
@@ -4894,6 +4931,16 @@ bool PortsOrch::addLag(string lag_alias, uint32_t spa_id, int32_t switch_id)
 {
     SWSS_LOG_ENTER();
 
+    auto lagport = m_portList.find(lag_alias);
```

Comment:
Please add some comments. I believe this is the scenario where removeBridgePort is not yet successful due to fdb flush right?